src/execution_plan/optimizations/reduce_count.c

```diff
@@ -233,7 +233,7 @@ void _reduceEdgeCount(ExecutionPlan *plan) {
 				// No change to current count, -[:none_existing]->
 				break;
 			default:
-				edges += _countRelationshipEdges(Graph_GetRelationMatrix(g, relType));
+				edges += GraphStatistics_EdgeCount(&g->stats, relType);
```

Comment:
Please change this from `GraphStatistics_EdgeCount(&g->stats, relType);` to:
`Graph_StatisticsEdgeCount(g, relType);`